lib/ui/pages/main/home/home.dart

```diff
@@ -1,21 +1,46 @@
 import 'package:chat_flutter/config/app_space.dart';
 import 'package:chat_flutter/ui/molecules/home/list_tile.dart';
+import 'package:chat_flutter/ui/pages/main/home/home_controller.dart';
 import 'package:flutter/material.dart';
 import 'package:chat_flutter/model/user.dart';
 import 'package:chat_flutter/model/group.dart';
 
 import 'package:chat_flutter/config/app_text_size.dart';
+import 'package:provider/provider.dart';
 
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    final _homeController = Provider.of<HomeController>(context);
     return SingleChildScrollView(
       child: Column(
         children: <Widget>[
-          HomePageListTile(
-            name: 'name',
-            imgUrl: 'https://dot.asahi.com/S2000/upload/2019100100055_1.jpg',
-            isMe: true,
+          FutureBuilder(
+            future: _homeController.getMeById('test'),
+            builder: (BuildContext context, AsyncSnapshot<User> snapshot) {
+              if (snapshot.hasError) {
+                return Center(child: Text('Error: ${snapshot.error}'));
+              }
+              switch (snapshot.connectionState) {
+                case ConnectionState.waiting: // データの取得まち
+                  return Center(
+                    child: CircularProgressIndicator(),
+                  );
+
+                default:
+                  if (snapshot.hasData) {
+                    return HomePageListTile(
+                      name: snapshot.data.name,
+                      imgUrl: snapshot.data.imgUrl,
+                      isMe: true,
+                    );
+                  } else {
+                    return Center(
+                      child: Text("該当するユーザーがいません"),
+                    );
```

Comment:
```suggestion
                    return const Center(
                      child: Text("該当するユーザーがいません"),
                    );
```
のように書けますかね？
[`const`](https://dart.dev/guides/language/language-tour#final-and-const)をつけると、コンパイル時に確定できるので、パフォーマンスが上がります
以下参考
>Use const widgets where possible. (This is equivalent to caching a widget and re-using it.)

https://medium.com/flutter-jp/state-performance-7a5f67d62edd